examples/basic-express-typescript/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "basic-express-typescript",
+  "version": "0.0.1",
+  "description": "Example of using TypeScript with fh-sync",
+  "main": "server.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "tsc && node server.js"
+  },
+  "author": "Evan Shortiss <evanshortiss@gmail.com> (http://evanshortiss.com/)",
```

Comment:
I'm not against that, but having personalized author in organization code seems to be antipatern.

Reply:
@wtrocki that's a mistake. `npm init` filled this in with my default and I forgot to remove 😊